shuup/core/templatetags/shuup_common.py

```diff
@@ -143,10 +143,18 @@ def safe_product_description(value):
     if isinstance(value, Undefined):
         return value
     if not settings.SHUUP_ADMIN_ALLOW_HTML_IN_PRODUCT_DESCRIPTION:
-        value = linebreaks(bleach.clean(value, tags=[]))
+        value = linebreaks(bleach.clean(value, tags=[], strip=True))
     return mark_safe(value)
 
 
+@library.filter
+def cleanmeta(value):
+    if isinstance(value, Undefined):
+        return value
+    prepared_content = value.replace("&nbsp;", " ").replace("</p>", "</p> ").replace("<br>", " ").replace('"', "'")
```

Comment:
Why you keep the p tag here ?

Reply:
because it will be removed by bleach anyways